requirements-dev.txt

```diff
@@ -30,3 +30,4 @@ scikit-learn>=0.20.0
 scikit-quant<=0.7;platform_system != 'Windows'
 jax;platform_system != 'Windows'
 jaxlib;platform_system != 'Windows'
+docplex
```

Comment:
@mtreinish At 697d37c, I've introduced a dependency to `docplex`. I thought `requirements-dev.txt` is the right place to add, but it causes [a test failure](https://github.com/Qiskit/qiskit-terra/runs/2872669446). What do you think is the best way to add `docplex` dependency?

Reply:
I suggest to add docplex in https://github.com/Qiskit/qiskit-terra/blob/62383eb90e1d3fc9790ceef9789e8f842e671a1e/azure-pipelines.yml#L817

Reply:
Based on @t-imamichi 's comment, I addressed this issue at 761745d. Could you check if the issue is fixed rightly?

Reply:
https://github.com/Qiskit/qiskit-terra/commit/761745d6c0ac5411ccfb872bcd87171a8449929c looks correct to me